apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/IFlushableConsumer.cs

```diff
@@ -23,5 +26,10 @@ internal interface IFlushableConsumer<T> : IConsumer<T>
         /// Flush the consumer.
         /// </summary>
         void Flush();
+
+        /// <summary>
+        /// Flush the consumer asynchronously.
+        /// </summary>
+        Task FlushAsync(CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Add return and parameter documentation, here and above. (Ideally change "flush" to "flushes" in both methods too, in the docs.)